File: flutter_embed_unity_platform_interface/lib/flutter_embed_constants.dart
class FlutterEmbedConstants {
  // A unique identifier used by the PlatformViewFactory to identify the Unity Player view,
  // and as the unique name of the method channel used for communication between Flutter and native
  // Should be the same value in flutter_embed_unity_android/android/src/main/kotlin/com/learntoflutter/flutter_embed_unity_android/flutter_embed_constants.kt
  static const uniqueIdentifier = "com.learntoflutter/flutter_embed_unity";
  static const methodNameSendToUnity = "sendToUnity";
  static const methodNameSendToFlutter = "sendToFlutter";
  static const methodNamePauseUnity = "pauseUnity";
  static const methodNameResumeUnity = "resumeUnity";
}

File: flutter_embed_unity_2022_3_android/android/src/main/kotlin/com/learntoflutter/flutter_embed_unity_android/constants/FlutterEmbedConstants.kt
package com.learntoflutter.flutter_embed_unity_android.constants

class FlutterEmbedConstants {
    companion object {
        const val logTag = "FlutterEmbedLog"
        // A unique identifier used by the PlatformViewFactory to identify the Unity Player view,
        // and as the unique name of the method channel used for communication between Flutter and native
        // Should be the same value in flutter_embed_unity/flutter_embed_unity_platform_interface/lib/flutter_embed_constants.dart
        const val uniqueIdentifier = "com.learntoflutter/flutter_embed_unity"
        const val methodNameSendToUnity = "sendToUnity"
        const val methodNameSendToFlutter = "sendToFlutter"
        const val methodNamePauseUnity = "pauseUnity"
        const val methodNameResumeUnity = "resumeUnity"
    }
}